packages/redux/src/helpers/buildQueryStringFromObject.ts

```diff
@@ -31,7 +31,8 @@ const buildQueryStringFromObject = (
     // which needs the `|` separating each different values of the same facet
     const parsedValue = Array.isArray(value) ? value.join('|') : value;
 
-    parsedValue && paramsToUrl.push([`${key.toLowerCase()}=${parsedValue}`]);
+    (parsedValue || typeof parsedValue === 'boolean') &&
```

Comment:
```suggestion
    parsedValue !== undefined &&
```

Seems simpler, and I believe it accomplishes the same goal 🙂

Reply:
I can't check if it's not undefined because it will allow empty strings and other falsy values that are defined